keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/DefaultBaseImageCache.kt

```diff
@@ -10,7 +10,7 @@ class DefaultBaseImageCache(
     baseImages[os]?.get(label.name.toLowerCase()) ?: throw UnknownBaseImage(os, label)
 }
 
-@ConfigurationProperties("keel.plugins.bakery.base-images")
+@ConfigurationProperties(prefix = "keel.plugins.bakery")
```

Comment:
D'oh. Thanks for the fix. However, we shouldn't actually be using this inside of Netflix.

Reply:
Hmm, ok. Let’s talk offline. We’re definitely calling methods that rely on this config.

Reply:
P.S. This is from #649 too.

Reply:
If you run oss keel, you'll need that. If you run -nflx keel, we will get the base amis.

Reply:
Rebased.

Reply:
Rebased.